infra/src/main/resources/db/migration/prod/V32__separate_segment_geometry.sql

```diff
@@ -5,7 +5,7 @@ create or replace function layout.calculate_geometry_hash(
   height_values numeric[],
   cant_values numeric[]
 ) returns uuid language sql as $$
-  select md5(row(resolution, geometry, height_values, cant_values)::text)::uuid
+  select md5(row(resolution, postgis.st_astext(geometry), height_values, cant_values)::text)::uuid
```

Comment:
Tämän seurauksena pitää lanata dev-ympäristön kanta koska tein muutoksen suoraan versioituun migraatioon. Silti helpompi näin kuin tehdä tätä toisena muutoksena, koska muutokset ei pitäisi olla mennyt vielä tuon pidemmälle.

Muutos korjaa siis sen että geometrian sisäinen esitys on liukulukuna tarkempi kuin ulos luettava muoto ja sitä kautta sen lukeminen ja uudelleenkirjoitus aiheuttaa tarpeettoman muutoksen. Nyt hash perustuu samaan kuin ulos luettava eikä pitäisi enää tapahtua. Poistaa noin 20% duplikaattidata geometrioista ja parantaa tuota initial metadatan linkitystä (se perustuu geometrian identtisyyteen)